src/python/requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 -r requirements.txt
 -r requirements-test.txt
+-r requirements-doc.txt
```

Comment:
Actually Yes, for documenting the package, we need to parse all modules (.py files) and some modules are importing foglamp